src/main/java/net/sally/chess/Board.java

```diff
@@ -6,17 +6,77 @@
 import java.util.List;
 
 public class Board {
-    private List<Pawn> pawns = new ArrayList<>();
+    private List<Pawn> whitePawns = new ArrayList<>();
+    private List<Pawn> blackPawns = new ArrayList<>();
 
-    public void addPawn(Pawn pawn) {
-        pawns.add(pawn);
+    public static int BOARD_SIZE = 8;
+
+    public void initialize() {
+        for (int i = 0; i < BOARD_SIZE; i++) {
+            addWhitePawn(new Pawn());
+            addBlackPawn(new Pawn(Pawn.BLACK, Pawn.BLACK_REPRESENTATION));
+        }
+    }
+
+    public String getWhitePawnsResult() {
+        return getPawnsResult(whitePawns);
+    }
+
+    public String getBlackPawnsResult() {
+        return getPawnsResult(blackPawns);
+    }
+
+    private String getPawnsResult(List<Pawn> pawns) {
+        StringBuilder pawnsResult = new StringBuilder();
+        for (Pawn p : pawns) {
+            pawnsResult.append(p.getRepresentation());
+        }
+        return pawnsResult.toString();
+    }
+
+    public void addWhitePawn(Pawn pawn) {
+        if (pawn.getColor().equals(Pawn.WHITE)) {
+            whitePawns.add(pawn);
+        }
     }
 
-    public int getPawnSize() {
-        return pawns.size();
+    public void addBlackPawn(Pawn pawn) {
+        if (pawn.getColor().equals(Pawn.BLACK)) {
+            blackPawns.add(pawn);
+        }
     }
 
-    public Pawn findPawn(int i) {
-        return pawns.get(i);
+    public int getWhitePawnSize() {
+        return whitePawns.size();
+    }
+
+    public int getBlackPawnsSize() {
+        return blackPawns.size();
+    }
+
+    public Pawn findWhitePawn(int i) {
+        return whitePawns.get(i);
+    }
+
+    public Pawn findBlackPawn(int i) {
+        return blackPawns.get(i);
+    }
+
+    public void print() {
+        StringBuilder printer = new StringBuilder();
+        printer.append("........\n");
+        printer.append("........\n");
+        printer.append(getBlackPawnsResult());
+        printer.append(System.lineSeparator());
+        printer.append("........\n");
+        printer.append("........\n");
+        printer.append("........\n");
+        printer.append("........\n");
+        printer.append("........\n");
+        printer.append(getWhitePawnsResult());
+        printer.append(System.lineSeparator());
+        printer.append("........");
```

Comment:
```suggestion
        StringBuilder printer = new StringBuilder();
        printer.append("........" + NEW_LINE);
        printer.append("........" + NEW_LINE);
        printer.append(getBlackPawnsResult());
        printer.append(NEW_LINE);
        printer.append("........" + NEW_LINE);
        printer.append("........" + NEW_LINE);
        printer.append("........" + NEW_LINE);
        printer.append("........" + NEW_LINE);
        printer.append(getWhitePawnsResult());
        printer.append(NEW_LINE);
        printer.append("........");
```
이렇게 상수로 선언해놓고 사용하는 방식을 의도했었습니다 ㅎㅎ
`private static final String NEW_LINE = System.lineSeparator();`

"....." 같은 경우도 현재로서는 변하지 않는 문자열 값이니 상수로 처리해서 사용하는게 좋겠죠?
"....."과 NEW_LINE을 합쳐서 상수로 선언해도 되겠네요:)

Reply:
앗... 사실 처음에 그렇게 상수로 지정했다가.. lineSeparator를 검색하다 이렇게 되었어요..🥲 다시 수정하였습니다!